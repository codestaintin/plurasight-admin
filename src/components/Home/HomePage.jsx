import React from 'react';
import { Link } from 'react-router-dom';
import Header from '../commons/Header';

/**
 *
 * @function HomePage
 *
 * @returns {JSX}
 */
const HomePage = () => (
  <React.Fragment>
    <div className="app jumbotron">
      <h1>Plurasight Administration</h1>
      <p>
						React, Redux and React Router in ES6 for ultra-responsive web apps
      </p>
      <Link to="/about" className="btn btn-primary btn-lg">Learn More</Link>
    </div>
  </React.Fragment>
);

export default HomePage;
