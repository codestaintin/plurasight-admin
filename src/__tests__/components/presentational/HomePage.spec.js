import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { configure, shallow } from 'enzyme';
import HomePage from '../../../components/Home/HomePage';

configure({ adapter: new Adapter() });

describe('<HomePage>', () => {
  it('renders the <HomePage> component', () => {
    const tree = shallow(<HomePage />);
    expect(tree).toMatchSnapshot();
  });
});
