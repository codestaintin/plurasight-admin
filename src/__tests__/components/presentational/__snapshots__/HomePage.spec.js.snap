// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage> renders the <HomePage> component 1`] = `
<Fragment>
  <div
    className="app jumbotron"
  >
    <h1>
      Plurasight Administration
    </h1>
    <p>
      React, Redux and React Router in ES6 for ultra-responsive web apps
    </p>
    <Link
      className="btn btn-primary btn-lg"
      replace={false}
      to="/about"
    >
      Learn More
    </Link>
  </div>
</Fragment>
`;
